app/(main)/settings/team/actions.ts

```diff
@@ -269,7 +255,6 @@ export async function updateTeamMemberRole(formData: FormData) {
 
 		return {
 			success: true,
-			isUpdatingSelf,
```

Comment:
`isUpdatingSelf` was removed because it was originally introduced for optimistic updates, but was not used as expected in the previous implementation, so it was removed to clean up state management.